ARE/components/processor.comport/src/main/java/eu/asterics/component/processor/comport/ComPortInstance.java

```diff
@@ -198,6 +211,24 @@ public Object setRuntimePropertyValue(String propertyName, Object newValue) {
         return null;
     }
 
+    final IRuntimeEventListenerPort elpEnablePlugin = new IRuntimeEventListenerPort() {
+        public void receiveEvent(final String data) {
+            if (!propEnabled) {
+                propEnabled = true;
```

Comment:
is propEnabled describes the state of an enabled (started) plugin, then I would suggest to 
1. start()
2. propEnabled=true;

Reply:
`start()` does nothing and returns immediately, if `propEnabled == false`. Therefore `propEnabled` has to be set to `true` before starting.